nextflow.config

```diff
@@ -179,8 +179,10 @@ singularity.registry = 'quay.io'
 // Nextflow plugins
 plugins {
     id 'nf-validation' // Validation of pipeline parameters and creation of an input channel from a sample sheet
```

Comment:
Could you change `nf-validation` to `nf-validation@1.1.3` so that we are using a fixed version.

Reply:
Will do!